package.json

```diff
@@ -115,12 +115,12 @@
 		"deep-freeze": "^0.0.1",
 		"dotenv": "^8.0.0",
 		"errorhandler": "^1.5.1",
-		"express": "4.17.0",
+		"express": "^4.21.0",
 		"express-flash": "0.0.2",
 		"express-session": "^1.16.2",
 		"express-validator": "^6.14.0",
 		"ini": ">=1.3.6",
-		"jsonwebtoken": "^8.5.1",
+		"jsonwebtoken": "^9.0.2",
```

Comment:
Breaking changes: https://github.com/auth0/node-jsonwebtoken/blob/master/CHANGELOG.md

Potential impact? Will have to assess. https://github.com/icgc-argo/argo-clinical/blob/develop/src/decorators/index.ts#L55

Reply:
So breaking changes:
- No longer supports Node 11 (not an issue)
- No longer supports unsigned tokens by default (shouldn't be an issue)
- RSA key must be >2048 bits, we are using RS256 so I believe this is fine